app/models/account.rb

```diff
@@ -3,10 +3,15 @@
 require_dependency 'backend_client'
 
 class Account < ApplicationRecord
+  # Hack to remove payment_gateway_type, payment_gateway_options and deleted_at from @attributes
+  # It is enough for rails not persisting them in actual columns.
   def self.columns
-    super.reject { |c| c.name == "deleted_at" }
+    super.reject{|column| /\Apayment_gateway_(type|options)|deleted_at\Z/ =~ column.name }
```

Comment:
I guess I am missing an important piece of information here but these 2 seem to be in the schema right now and I don't know if we have planned to remove them of why 🤔
https://github.com/3scale/porta/blob/d6a617f595fd519fc85b5d02ff94223d6a1b5445/db/schema.rb#L44-L45

Reply:
Sure we need to remove them one day

Reply:
I don't get why enforcing these 2 columns here but I see it is not newly introduced by you so I'll consider it out of scope 😄